src/event.rs

```diff
@@ -245,6 +245,12 @@ pub enum WindowEvent {
         /// [`transform`]: https://developer.mozilla.org/en-US/docs/Web/CSS/transform
         position: PhysicalPosition<f64>,
 
+        /// Indicates whether the event is created by a primary pointer.
+        ///
+        /// A pointer is considered primary when it's a mouse, the first finger in a multi-touch
+        /// interaction, or an unknown pointer source.
+        primary: bool,
```

Comment:
Would it make sense to link to https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/isPrimary as a reference for this?

Reply:
I don't think, since it's present on a lot of platforms. At least one could follow up if there will be questions, but linking in docs to external resources when we can explain ourselves is a bit to much.

Reply:
Something called "primary" is _present_ on many platforms, but we implement the _semantics_ of the web, which is why I thought it useful.

Reply:
I don't think there's a web semantic, web also forwards platform primary information forward.

I can add, but I think it's pretty clear right now, since it states basically the same as the link, without confusing parts.